slopmop/utils/environment.py

```
@@ -0,0 +1,40 @@
+    return False
+
+
+def is_interactive_terminal() -> bool:
+    """Return True if stdout is a TTY and not an agent/CI environment."""
+    if not sys.stdout.isatty():
+        return False
+    if is_agent_environment():
+        return False
+    if os.environ.get("NO_COLOR"):
+        return False
+    return True
```

Ruff (0.15.15)

[warning] Return the condition `not os.environ.get("NO_COLOR")` directly

Replace with `return not os.environ.get("NO_COLOR")`

(SIM103)